practitioner-service/src/main/java/uk/ac/ox/ndph/mts/practitioner_service/service/PractitionerService.java

```diff
@@ -46,8 +71,12 @@ public String savePractitioner(Practitioner practitioner) {
     }
 
     private void validateArgument(String value, String argumentName) {
```

Comment:
Create a validation class - not sure if the service should be concerned about the dynamics of the validation?